packages/discovery-provider/src/tasks/entity_manager/entities/user.py

```diff
@@ -345,6 +370,11 @@ def update_user_metadata(user_record: User, metadata: Dict, params):
     if "events" in metadata and metadata["events"]:
         update_user_events(user_record, metadata["events"], challenge_event_bus, params)
 
+    if "spl_usdc_payout_wallet" in metadata:
```

Comment:
nit: Had a moment of confusion here. Is `usdc` redundant here? This is an actual wallet address, right? Not a token account address? Would we presumably let users set different wallets for payout of various token types?

Reply:
Fantastic question...

cc: @raymondjacobson 

What do you think about storing the token account address, and on the UI, getting the owner of that address to show to the user?

Reply:
I agree @rickyrombo . I actually was expecting us to store the USDC token account rather than the wallet, but always showing the owner to the user. My one test account on stage has the token account stored in this value. That way it's harder to mess up if we shove it into something that expects to do USDC. open to other thoughts

Reply:
perf, aligned

Reply:
late to respond to this: But I think the general paradigm across experiences is to store the wallet, not the account. It's trivial to lookup the account through code, less so for users to find the account address for a specific token and paste that in.